.buildkite/scripts/promote_launchers.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# If the user specified a fully-qualified identifier for
+# `core/hab-launcher` in the Buildkite metadata, then promote that
+# release to the release channel.
+#
+# Otherwise, promote the latest stable release of the Launcher to the
+# release channel.
+
+set -euo pipefail
+source .buildkite/scripts/shared.sh
+
+resolve_launcher() {
+    local metadata_key="${1}"
+    local target="${2}"
+
+    if buildkite-agent meta-data exists "${metadata_key}"; then
+        launcher_ident=$(buildkite-agent meta-data get "${metadata_key}")
+    else
+        launcher_ident=$(latest_from_builder "${target}" stable hab-launcher)
+    fi
+    echo "${launcher_ident}"
```

Comment:
There's no need for the command substitution and `echo`. This could be simplified to:
```bash
    if buildkite-agent meta-data exists "${metadata_key}"; then
        buildkite-agent meta-data get "${metadata_key}"
    else
        latest_from_builder "${target}" stable hab-launcher
    fi
```

Reply:
Ah, nice catch... will change!